### DISCUSSION TITLE
Orography

### DISCUSSION THREAD

**Discussion by feggleton** [cf-convention/discuss#352] [Q&A about using CF] [answered]
### Question

How do you describe orography in CF metadata? We have a model which will be producing rainfall rate on smoothed and unsmoothed orography, we need to show that these are different in the metadata. How do you describe this information in the metadata?

**Comment by taylor13** [comment, marked as answer]
I don't think there is a standard way to do this.  A similar problem is describing rainfall rate in two different simulations.  There is no standard way in CF to indicate the rainfall is produced under control run conditions vs. some perturbed run conditions.  You would be free, of course, in defining your own attribute for this purpose.  Or you could include the information in the "source" attribute or the "long_name", where there are no controlled vocabularies imposed.  
There are other ways you could include it by defining a "orography type" coordinate variable, but I probably wouldn't do that.

**Reply by feggleton** [reply]
Thanks Karl, I couldn't find anything so suspected there wasn't a standard way. Currently looking like we would go down the long_name route but it's not ideal. Now to look at how we represent this in GRIB!

**Comment by ChrisBarker-NOAA** [comment]
This is a challenge of standard names -- they can describe "what" a variable is, but there can be any number of ways that that data could have been generated. 

Think of the standard name of "latitude" for instance -- it's clear that it's a latitude but the latitude of what? For teh most part, that is determined by some other aspect of the meta-data.

It's a challenge where to draw the line -- I have no opinion about where it belongs in this case.
